### ISSUE TITLE
feat: media-embedding in messages

### ISSUE THREAD

**Issue by MichaelMure** [MichaelMure/git-bug#10]
As we can store arbitrary data in the git storage and reference it in a commit, we can support media in bug's comment.

In particular, we can have:

- images (jpg, png, gif ..)
- video (as supported by major browser)
- arbitrary files

Markdown support images but not video (though gitlab reuse the same syntax: https://gitlab.com/help/user/markdown#videos), nor arbitrary files.

On top of that, as files would not be in the normal repository, we need a way to reference the corresponding blob.

A way to do that might be:

```markdown
Inline-style:
![alt text](e69de29bb2d1d6434b8b29ae775ad8c2e48c5391 "logo.png")

Reference-style:
![alt text1][logo]

[logo]: e69de29bb2d1d6434b8b29ae775ad8c2e48c5391 "logo.png"
```
Additionally, git-bug could enforce the reference style to make it more readable for terminal users.

**Comment by nkgm** [comment]
Would it be possible to use the webui to paste an image? (begging the additional question of using the webui for issue creation/editing)

**Comment by MichaelMure** [comment]
@nkgm yes, that would be totally neat. I'd love to have the same UX as on github.

**Comment by keith-hall** [comment]
Seeing as this feature is on the planned feature matrix at https://github.com/MichaelMure/git-bug/blob/master/doc/feature_matrix.md#entities, I don't think it makes sense to make it stale or close it...

**Comment by sudoforge** [comment]
@keith-hall the stale bot was added to aid in pruning issues and pull requests that have not had activity in a while, given the slowdown of development over the past few years. Indeed, this is an issue that is definitely a desirable feature.

**Comment by MichaelMure** [comment]
Not stale

**Comment by nh2** [comment]
I synced issues from Github and found all images and videos to be missing.

Is this issue the one that tracks that, or should that already be working?
